src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age && (
+      <p className="Person__age">{`I am ${person.age}`}</p>
+    )}
+    {person.isMarried ? (
+      <p className="Person__partner">
+        {`${person.partnerName} is my `}
+        {person.sex === 'm' ? ('wife') : ('husband')}
```

Comment:
using nested ternary operator is a bad practice, it's better to move it to the variable